_config.space.scss

```diff
@@ -0,0 +1,5 @@
+$space: 16px; // Default AP space variable
+
+// Bulma spacing
+
+$column-gap: $space/2;
```

Comment:
I think we have different gaps between columns depending on where we're using them because we use the bulma columns for different modules with columnar layouts... idk how we address that 😕

Reply:
Have discussed this with @beccadawson and Lizzie and think we are in agreement that for now leaving this as is. Once we have figured out how to combat the issue on media centre we can port that solution onto the insurance section of the site